packages/vscode-extension/package.json

```diff
@@ -807,6 +807,12 @@
             "type": "string",
             "description": "Set the default root directory for creating new teams app project. By default, the root directory is ${homeDir}/TeamsApps. (requries reload of VS Code)",
             "default": "${homeDir}/TeamsApps"
+          },
+          "fx-extension.automaticNpmInstall": {
```

Comment:
Any way to hide/disable this feature before releasing *validate-prerequisites* in `tasks.json`.
Current F5 still runs `npm install` which is a bit conflict with this feature.

Reply:
Removing the setting here can simply disable this feature. Let me remove it.